spec/lib/reporting/cloudwatch_client_spec.rb

```diff
@@ -233,6 +233,9 @@ def stub_single_page
             ),
           },
         }
+
+        # override Zonebie
+        allow(Time).to receive(:zone).and_return(nil)
```

Comment:
Does this imply zonebie is calling `Time.zone` itself? No issue wit hit review-wise, just curious.

Reply:
I tried setting `Time.zone = nil` in a before filter, in the spec itself, etc, and they all got overridden, so I had to stub to get the spec to actually run with a nil timezone